README.md

```diff
@@ -60,30 +60,30 @@ git clone https://github.com/KELiON/cerebro.git cerebro
 And then install dependencies.
 
 ```bash
-$ cd cerebro && npm install && cd ./app && npm install && cd ../
+$ cd cerebro && yarn && cd ./app && yarn && cd ../
 ```
 
 ### Run
 ```bash
-$ npm run dev
+$ yarn dev
```

Comment:
@Kageetai hm, I also can't find it in docs now and don't remember where I found it. But yes, `yarn dev` === `yarn run dev`. Since it is not documented I'll better replace it with `yarn run`